Source/Core/DolphinWX/Config/GeneralConfigPane.cpp

```diff
@@ -198,7 +199,8 @@ void GeneralConfigPane::OnDualCoreCheckBoxChanged(wxCommandEvent& event)
   if (Core::IsRunning())
     return;
 
-  SConfig::GetInstance().bCPUThread = m_dual_core_checkbox->IsChecked();
+  Config::SetBase(Config::CPU_THREAD, m_dual_core_checkbox->IsChecked());
+  Config::InvokeConfigChangedCallbacks();
```

Comment:
Need not mean that I know better, I just wanted to put it out there in case someone who does wants to chime in.
Let's just keep it like that, and change it later if necessary ;)